### ISSUE TITLE
Custom field

### ISSUE THREAD

**Issue by meugamer** [mtoensing/wp-tgdb-importer#2]
Hello, I really liked your project. And I tested the same on my site. But I have a question the custom fields do not appear in the post. There is some code that is needed. Do I have to use acf to put the fields appear in the post?

**Comment by mtoensing** [comment]
No, the custom fields should be visible on the imported custom post types. Maybe you disabled the visibility of the custom field panel in wp backend?

**Comment by meugamer** [comment]
Here it appears as active in the post field. The problem is that I have tested up the standard wordpress layouts and it also does not work. But when I manually put the code in the loop it appears but has no way to control it. In the post case, I specify. He does not appear.

**Comment by mtoensing** [comment]
This plugin only attaches the data to the game custom post type. You have to access the data yourself. See here https://codex.wordpress.org/Custom_Fields
